scipy/stats/_distn_infrastructure.py

```diff
@@ -3140,13 +3137,23 @@ def expect(self, func=None, args=(), loc=0, lb=None, ub=None,
             argument (parameters) of the distribution
         lb, ub : numbers, optional
             lower and upper bound for integration, default is set to the
-            support of the distribution, lb and ub are inclusive (ul<=k<=ub)
+            support of the distribution, lb and ub are inclusive
+            (``ul <= k <= ub``).
         conditional : bool, optional
             Default is False.
             If true then the expectation is corrected by the conditional
             probability of the integration interval. The return value is the
             expectation of the function, conditional on being in the given
-            interval (k such that ul<=k<=ub).
+            interval (``k`` such that ``ul <= k <= ub``).
+        kwds : additional keywords to control the summation.
+
+            - maxcount : maximal number of terms to evaluate
+                        (to avoid an endless loop for an infinite sum).
+                        Default is a 1000.
+            - tolerance : absolute tolerance for the summation.
+                          Default is 1e-10.
+            - chunksize : iterate over the support of a distributions in chunks
+                          of this size. Default is 48.
```

Comment:
I don't understand why these are defined *and advertised* as part of `kwds`, rather than as separate keyword arguments with default values in the function definition.